starknet_devnet/blueprints/rpc/structures/responses.py

```diff
@@ -176,15 +175,11 @@ def events() -> List[Event]:
             _events.append(event)
         return _events
 
-    def status() -> str:
-        if txr.status is None:
-            return "UNKNOWN"
-
-        mapping = {
-            TransactionStatus.NOT_RECEIVED: "UNKNOWN",
+    def status() -> TxnStatus:
+        mapping: Dict[TransactionStatus, TxnStatus] = {
             TransactionStatus.ACCEPTED_ON_L2: "ACCEPTED_ON_L2",
             TransactionStatus.ACCEPTED_ON_L1: "ACCEPTED_ON_L1",
-            TransactionStatus.RECEIVED: "RECEIVED",
+            TransactionStatus.RECEIVED: "PENDING",
```

Comment:
Hmm, I do not like this one :/ Does spec define it this way?

Reply:
RPC 0.2.1 spec only defines these
```json
"TXN_STATUS": {
    "type": "string",
    "enum": [
        "PENDING",
        "ACCEPTED_ON_L2",
        "ACCEPTED_ON_L1",
        "REJECTED"
    ],
    "description": "The status of the transaction"
}
```